src/main/java/com/dougnoel/sentinel/filemanagers/FileManager.java

```diff
@@ -98,4 +104,60 @@ public static String getClassPath(String className) {
 			return null;
 		}
 	}
+	
+	/**
+	 * Saves an image of type File to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the sub-directory is null, the root image directory will be used.
```

Comment:
This line should go in the description above.

Reply:
Done